lib/utils/fs.go

```diff
@@ -154,16 +154,20 @@ func StatDir(path string) (os.FileInfo, error) {
 	return fi, nil
 }
 
+func getPlatformLockFilePath(path string) string {
```

Comment:
move the function to the `_unix.go` and `_windows.go` files so the function is closer to `lockPostFix`(unix does not need it if the function is defined local).
And merge it!

Reply:
Done 👍